stmt.go

```diff
@@ -38,15 +39,15 @@ func (s *stmt) NumInput() int {
 }
 
 func (s *stmt) Exec(args []driver.Value) (driver.Result, error) {
-	panic("Using ExecContext instead")
+	return nil, fmt.Errorf("use ExecContext instead")
```

Comment:
Nits: Is there any way that we can call `ExecContext` here and convert `[]driver.Value` to `[]driver.NamedValue`?

Reply:
It might be technically possible, but it is not necessary, as the Go sql driver package will handle this. The documentation states that this method is no longer in use: https://pkg.go.dev/database/sql/driver#Execer